dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -172,83 +172,91 @@ def release_management():
 
 
 @release_management.command(
-    name="prepare-airflow-package",
-    help="Prepare sdist/whl package of Airflow.",
+    name="prepare-provider-documentation",
+    help="Prepare CHANGELOG, README and COMMITS information for providers.",
 )
-@option_package_format
-@option_version_suffix_for_pypi
 @option_debug_release_management
+@argument_packages
+@click.option(
+    "--base-branch",
+    type=str,
+    default="main",
+    help="Base branch to use as diff for documentation generation (used for releasing from old branch)",
+)
 @option_github_repository
+@click.option(
+    "--only-min-version-update",
```

Comment:
cc: @eladkal -> when you use this flag, the only thing tha will happen is regeneration of the files where min-airflow-version is set.

Reply:
This is very cool!